### ISSUE TITLE
English translation?

### ISSUE THREAD

**Issue by EAGBCD** [ET-Team/EnigTech2#562]
Hello! 

I hope you understand the English I am typing in. 

Your modpack seems amazing, and just what I have been looking for for a long time. 

I can see how much work you have put into it and I apologize for making demands for you to do extra work. 

Will the English translation ever be completed? I download the pack and looked through the advancements and only 50% are in English, this stops me from playing this amazing pack because I know I will get frustrated, when the English quests end, and I can no longer understand what to do next. 

Basically I am wondering if the English for the rest of the pack is still being worked on or if you have given up? Or if there is anything I can do to get the full pack in English so I can understand it. 

Either way, fantastic work, this is the exact direction I wanted to see modpacks going in, I wish you were an American so I could play it now. 

Just let me know the future of English for the pack, I would love to be able to play and read the whole thing. 

Thank you very much.

**Comment by dong031001** [comment]
Actually, the problem with completing the English translation of the modpack is lack of the time for the team members, who are mostly unavailable yet due to college work. However, as the Christmas vacation is coming near, I, per se, will have some time to work on the translation further. However, the English translation for the first three stages, I believe, was completed, and there should be no problem for you to have a wonderful playthrough of them.

Thank you for your support tho, wish you a good day.

**Comment by EAGBCD** [comment]
Thank you, and the whole team for all the work put in to this modpack, it truly shows. Also thank you for the response.

 I suppose you are correct, I could start now, and hope that when I finish stage 3 more translations are available. As you said with Christmas vacation, there will be more time for it to be worked on. I am very glad that you plan to continue the English. I was worried that If I started now, I would play up to stage 3, and then have to quit and loose my hard work because I can no longer understand how to progress. Christmas is soon, so if you and others are planing to add to the English maybe I don't have to worry and can start now. By the time I finish stage 3 I think Christmas vacation would be over. 

If there are ways I can help, feel free to let me know, I feel bad just asking you to do things for me while I am putting nothing in but demands.

Well I think I'll listen to you and start playing, hopefully the timing can work out and by the time I need them, more translations are available. I looked through all the advancements and seems like it's at 50% English right now. 

Again, thank you very much, I appreciate the well made pack, the response, and your willingness to make it playable for me all the way across the world.

**Comment by EAGBCD** [comment]
Hello again! Sorry to bother you, I wanted to come back and report that I had a fantastic time completing the first 2 stages. Like I thought this is the pack I had been looking for, it is extremely fun and well made. Thank you for working so hard to make a modpack like this, the hard work shows at every second of gameplay. 

In my creative test world I looked at the future chapters to see if I could complete them without the translation. Unfortunately, I really can not, there are so many changes and custom things I need to read about what is going on.

The good news is its only a few stages left that are not translated, and some chapters even have a few english quests. 

So now that I can no longer play much I wanted ask how the final translations are progressing? It's understandable if you would rather work on other things, but I hope that I can keep playing this pack, especially after all the progress I made. It's really all up to you, I can't do much but ask and hope. 

Also in my testing I found all the names of the modular machines are not translated, this would make it very hard to use any of them because I can't tell the machine name, so  don't know what machines recipes are asking for, or how to tell the machines apart. 

If you do decide to complete the translations the machine blueprints would be an important part of that I feel. 

I also am confused about mining in this pack. Ores only spawn in big veins far apart form other veins, it makes it hard to find the ores I need. I saw the wiki and JEI can show the dimension and Y level, but I still find it very hard to find ores, and even harder to find the ones I need. How are people supposed to find ores they need without boring hours of searching? The scanner doesn't work to well from the surface, and it won't tell you what ore it found. 

Is there a discord for this pack where I can ask questions?


Either way thank you again, I hope I can keep play soon. Your decision though, I know you and the team are busy, and I am a small priority.

**Comment by dong031001** [comment]
> @dong031001
> 
> Hello again! Sorry to bother you, I wanted to come back and report that I had a fantastic time completing the first 2 stages. Like I thought this is the pack I had been looking for, it is extremely fun and well made. Thank you for working so hard to make a modpack like this, the hard work shows at every second of gameplay.
> 
> In my creative test world I looked at the future chapters to see if I could complete them without the translation. Unfortunately, I really can not, there are so many changes and custom things I need to read about what is going on.
> 
> The good news is its only a few stages left that are not translated, and some chapters even have a few english quests.
> 
> So now that I can no longer play much I wanted ask how the final translations are progressing? It's understandable if you would rather work on other things, but I hope that I can keep playing this pack, especially after all the progress I made. It's really all up to you, I can't do much but ask and hope.
> 
> Also in my testing I found all the names of the modular machines are not translated, this would make it very hard to use any of them because I can't tell the machine name, so don't know what machines recipes are asking for, or how to tell the machines apart.
> 
> If you do decide to complete the translations the machine blueprints would be an important part of that I feel.
> 
> I also am confused about mining in this pack. Ores only spawn in big veins far apart form other veins, it makes it hard to find the ores I need. I saw the wiki and JEI can show the dimension and Y level, but I still find it very hard to find ores, and even harder to find the ones I need. How are people supposed to find ores they need without boring hours of searching? The scanner doesn't work to well from the surface, and it won't tell you what ore it found.
> 
> Is there a discord for this pack where I can ask questions?
> 
> Either way thank you again, I hope I can keep play soon. Your decision though, I know you and the team are busy, and I am a small priority.

You can join our discord at any time, but it is still under construction. The link is here: https://discord.gg/AFqxvyRz

For the internationalization issue, the text had been sent to another translation guy, whose progress I think had overpassed 50%. Yet, unfortunately, I think that the names for modular machines are not currently supporting localization, i.e., we would not be able to translate those machines into English.

In terms of your mining issue, most of the ore veins that you were looking for have something we called "indicator," which are basically glowing roses on the ground. Find the roses and mine right down will lead you to the ore veins you were having a hard time finding. The "indicators" are also shown in the ore vein wiki in this repository.

Thanks for asking, feel free to ask any question in our discord channel and have a good day.

**Comment by EAGBCD** [comment]
Thank you again for the response and information.

The modular machine issue you bring up is very unfortunate, if it's true there is no way to support other languages for modular machine names it makes them very difficult to use. I can't even type the blueprint names on my keyboard. A user will have a hard time figuring out what machine a quest is talking about, have a hard time finding a description of that machine, and have a hard time knowing what the machine to use, or the differences between them. We can Use JEI to find recipes and uses but thats the extent of it. 

Not your fault obviously, I guess if you are investing time and effort into a full translation, you will need to figure something out so users can use and understand the modular machines. It's such a big focus of this modpack that you really can not ignore it.   

Again thank you for all the work and answers to my questions, I'll keep my eyes here for updates, and we can talk on discord too if you are on there.

**Comment by dong031001** [comment]
All Advancements are translated by now.
